mempool/src/core_mempool/transaction_store.rs

```diff
@@ -88,7 +89,8 @@ impl TransactionStore {
                 Duration::from_secs(t.txn.expiration_timestamp_secs())
             })),
             priority_index: PriorityIndex::new(),
-            timeline_index: TimelineIndex::new(),
+            timeline_index: MultiBucketTimelineIndex::new(config.broadcast_buckets.clone())
+                .expect("MultiBucketTimelineIndex must be initialized"),
```

Comment:
this removes all the error messages from MultiBucketTimelineIndex

either do unwrap, or if you want additional message .unwrap_or_else(|e| panic!("MultiBucketTimelineIndex must be initialized, {:?}", e))